sample_trial/services/init-service/application-dev.yml

```diff
@@ -33,12 +33,23 @@ trial:
       roles:
         - superuser
       userAccount: dc32d94e-1ade-4968-b83e-bab949b7e90c
+    - givenName: qacreate
+      familyName: qaadmin
+      prefix: Mr
+      roles:
+        - admin
+      userAccount: d4b73ba8-9d76-408b-b6e1-e3b8953b39e7
```

Comment:
Have you considered how the init service is going to use this new role? Looks like it's going to fail on the CI run...

Reply:
This new role isnt used by init-service so it wont fail. I added teh role to allow QA to test negative scenarios - i.e a role that doesnt have assign-role permissions.

Reply:
Liza added 3 AD users for auth, 2 of those users are bootstrap and automation which both have a superuser role that has access to all permissions. I added a qacreate user to  have the remaining AD userAccount and also NOT have all roles/permissions to allow negative testing - Ive spoken with Ben on this issue and he understands and agrees with the approach. He said that a possible tidy of the application.yml properties will be needed post dev testing.